yarn-project/simulator/src/client/private_execution.test.ts

```diff
@@ -868,41 +867,6 @@ describe('Private Execution test suite', () => {
         ).rejects.toThrow('Message not in state');
       });
     });
-
-    it('Should be able to consume a dummy public to private message', async () => {
```

Comment:
Another non-sensical test. It does nothing with messaging!

This test case was first created 20 months ago and I think it has been updated like 426 times in a negligent way that it became something completely different. So I nuked it